packages/salesforcedx-vscode-apex/coverconfig.json

```diff
@@ -4,6 +4,6 @@
   "relativeCoverageDir": "../../coverage",
   "ignorePatterns": ["**/node_modules/**"],
   "includePid": false,
-  "reports": ["json", "html", "lcov"],
+  "reports": ["json", "lcov"],
```

Comment:
I've removed the HTML coverage since they don't seem useful. They are also not used by codecov.